fastlane/metadata/android/en/changelogs/52.txt

```diff
@@ -0,0 +1 @@
+* Fixed downloads not working
```

Comment:
F-Droid falls back to en-US for fastlane metadata. So any translation tool needs to support this. "en" is most likely not going to work in this place.